server/src/items/classes/spell_scrolls/WardSpellScroll.js

```diff
@@ -0,0 +1,26 @@
+const SpellScroll = require("./SpellScroll");
+
+class WardSpellScroll extends SpellScroll {
+    onUsed() {
+        
+        console.log("Ward!");
+        const me = this;
```

Comment:
I think this outer scope variable can be avoided by using arrow functions inside the loop below, instead of the unnamed function expression.